lib/email_auth.py

```diff
@@ -0,0 +1,62 @@
+# email_auth.py
+#
+# Combine standard login with option of OAuth2.0
+#
+
+import smtplib
+import base64
+from httplib2 import Http
+from oauth2client import file, client, tools
+from googleapiclient.discovery import build
+from lib.config import open_config
+from imaplib import IMAP4_SSL
+
+config = open_config()
```

Comment:
let's make these in ALL_CAPS since they're constants, please

Reply:
fixed